controllers/assignments.py

```diff
@@ -529,13 +528,13 @@ def _autograde_one_q(course_name, assignment_id, sid, qname, points, deadline=No
     question = db(db.questions.name == qname).select().first()
 
     # dispatch on grading_type; if none specified, can't autograde
-    if question.autograde == 'unittest':
+    if autograde == 'unittest':
```

Comment:
I think I may have been confused about the recent conversation or maybe I'm confused by the values associated with the autograde column.

1.  Whether or not a question can be autograded is one thing.  I can see where this would be useful at the assignment level.  If you try to autograde an entire assignment there might be questions that are autogradeable and questions that are not.   But, if a question is autogradeable at all isn't it always autogradeable?

2.  How a question is autograded seems like a separate idea, and does not seem likely to change as the autograde method is kind of encompassed by the kind of question it is in the first place.